chapters/open_research.md

```diff
@@ -11,58 +11,57 @@ Recommended skill level: low.
-5. [Open hardware](#Open_hardware)
-    1. [Why open hardware?](#Why_open_hardware)
-    2. [Elements of an open source hardware project](#Elements_of_an_open_source_hardware_project)
-    3. [Open source hardware processes and practices](#Open_source_hardware_processes_and_practices)
-        1. [Designing your hardware](#Designing_your_hardware)
-        2. [Hosting your design file](#Hosting_your_design_files)
-        3. [Distributing open source hardware](#Distributing_open_source_hardware)
-6. [Open access](#Open_access)
-    1. [What is open access?](#What_is_open_access)
-        1. [Repositories and self-archiving](#Repositories_and_self_archiving)
-        2. [Open access publishing](#Open_access_publishing)
-    2. [Why does open access matter?](#Why_does_open_access_matter)
-    3. [Best practice for open access](#Best_practice_for_open_access)
-        1. [Self-archiving](#Self_archiving)
-        2. [Publication](#Publication)
-7. [Open notebooks](#Open_notebooks)
-8. [Open scholarship](#Open_scholarship)
-    1. [Open educational resources](#Open_educational_resources)
-    2. [Equity, Diversity, Inclusion](#Equity_Diversity_Inclusion)
-    3. [Citizen science](#Citizen_science)
-9. [Checklists](#Checklists)
-10. [What to learn next](#What_to_learn_next)
-11. [Further reading](#Further_reading)
-12. [Definitions/glossary](#Glossary)
-13. [Bibliography](#Bibliography)
-
-<a name="Summary"></a>
+1. [Summary](#summary)
+2. [How this will help you/ why this is useful](#why-this-is-useful)
+3. [Open data](#open-data)
+    1. [Steps to make your data open](#steps-to-make-your-data-open)
+        1. [Step 1: Make your data available](#step-1-make-your-data-available)
+        2. [Step 2: Make your data easy to understand](#step-2-make-your-data-easy-to-understand)
+        3. [Step 3: Make your data easy to use](#step-3-make-your-data-easy-to-use)
+        4. [Step 4: Make your data citeable](#step-4-make-your-data-citeable)
+    2. [Barriers to data sharing](#barriers-to-data-sharing)
+        1. [Privacy](#privacy)
+        2. [National and commercially sensitive data](#national-and-commercially-sensitive-data)
+4. [Open source software](#open-source-software)
+    1. [What is open source software?](#what-is-open-source-software)
+    2. [How running and contributing to open source software projects benefits you](#how-running-and-contributing-to-open-source-software-projects-benefits-you)
+        1. [Making your own work open source](#making-your-own-work-open-source)
+        2. [Contributing to others' open source software projects](#contributing-to-others-open-source-software-projects)
+    3. [How open source software benefits research](#how-open-source-software-benefits-research)
+    4. [How to run your own open source software project](#how-to-run-your-own-open-source-software-project)
+        1. [Readme](#readme)
```

Comment:
This isn't working for me....I wonder if README is a special word for GitHub?

Maybe re-name the heading to "Welcome users by adding information to your README"?

What do you think?